django_coverage_plugin/plugin.py

```diff
@@ -162,6 +163,13 @@ def file_tracer(self, filename):
     def file_reporter(self, filename):
         return FileReporter(filename)
 
+    def find_executable_files(self, src_dir):
+        for i, (dirpath, dirnames, filenames) in enumerate(os.walk(src_dir)):
```

Comment:
I don't think there's any reason to use `enumerate` here if you're not using `i` later on.